webapp/components/TeaserImage/TeaserImage.vue

```diff
@@ -178,7 +195,17 @@ export default {
   border-top: $border-size-base solid $border-color-softest;
 }
 
-.contribution-image {
-  max-height: 300px;
+.crop-overlay {
+  max-height: 2000px;
+  position: relative;
+  width: 100%;
+  background-color: #000;
+}
+
+.crop-confirm {
+  position: absolute;
+  left: 10px;
+  top: 10px;
+  z-index: 9999;
```

Comment:
```suggestion
  z-index: 1;
```

this puts the button on top of the overlay but under the header ;)